### PR TITLE
Fixed issue #15310: Display dropdown boxes - Date/Time question type not working correctly if minimum date < 1900

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/helpers/qanda_helper.php [hunk 1]
```diff
@@ -1003,12 +1003,12 @@ function do_date($ia)
                     * expressions are not supported because contents of dropbox cannot be easily updated dynamically
                     */
                     $yearmin = (int) substr($mindate, 0, 4);
-                    if (!isset($yearmin) || $yearmin < 1900 || $yearmin > 2037) {
+                    if (empty($yearmin)) {
```
Comment (Shnoulle) [inline application/helpers/qanda_helper.php:1006]: Must test with 1001 and 20 ?

**Review by Shnoulle — APPROVED** [review]

**Comment by gabrieljenik** [comment]
https://bugs.limesurvey.org/view.php?id=15310#c72174

**Comment by olleharstedt** [comment]
Wait. Was this not ready?

**Comment by gabrieljenik** [comment]
> Wait. Was this not ready?

This was kind of pending to be reviewed.
https://bugs.limesurvey.org/view.php?id=15310#c72174

**Comment by olleharstedt** [comment]
OK, maybe remove "Code review done" next time? I was too quick... :(

**Comment by gabrieljenik** [comment]
Yes, maybe we should change the color of the "desicion needed" label

**Pull request merged**